tests/acceptance/features/apiDownloads/download.feature

```diff
@@ -60,7 +60,15 @@ Feature: Download file in project space
       | Brian |
 
 
-  Scenario: user with role viewer cannot get the old version of the file in the project space
+  Scenario: user with role viewer cannot download the old version of the file in the project space
     Given user "Alice" has uploaded a file inside space "download file" with content "new content" to "file.txt"
-    When user "Bob" tries to get version of the file "file.txt" with the index "1" of the space "download file" using the WebDAV API
+    And user "Alice" has uploaded a file inside space "download file" with content "newest content" to "file.txt"
+    When user "Bob" tries to download version of the file "file.txt" with the index "1" of the space "download file" using the WebDAV API
```

Comment:
can we merge both actions in one scenario?
```suggestion
    And user "Alice" has uploaded a file inside space "download file" with content "newest content" to "file.txt"
    When user "Bob" tries to get versions of the file "file.txt" from the space "download file" using the WebDAV API
    Then the HTTP status code should be "403"
    When user "Bob" tries to download version of the file "file.txt" with the index "1" of the space "download file" using the WebDAV API
```

Reply:
It is possible. Is that common practice now in ocis API tests, to reduce run time?

Reply:
Yeah, I think it's possible. What would you suggest for the scenario title?
Is this fine ? : 
Scenario: user with role viewer cannot download the old version and get versions of the file in the project space

Reply:
or may be :
Scenario: user with role viewer cannot get a list of versions and cannot download the old version of the file in the project space

Reply:
> It is possible. Is that common practice now in ocis API tests, to reduce run time?

since the actions are similar, I would test them in one scenario. this can reduce run time as well.

Reply:
> Yeah, I think it's possible. What would you suggest for the scenario title? Is this fine ? : Scenario: user with role viewer cannot download the old version and get versions of the file in the project space

`Scenario: user with role viewer cannot get the versions of a file in the project space`: this should be fine

Reply:
Okay. Will update it right now.